File: Latihan B/Java/SoccerClub.java
public class SoccerClub {

    /* Private Attribute */
    private String clubName;
    private String clubOrigin;
    private String clubEst;

    /* Constuctor */
    public SoccerClub(String clubName, String clubOrigin, String clubEst) {
        this.clubName = clubName;
        this.clubOrigin = clubOrigin;
        this.clubEst = clubEst;
    }

    public SoccerClub() {

    }

    /* Setter dan Getter Club Name */
    public void setClubName(String clubName) {
        this.clubName = clubName;
    }

    public String getClubName() {
        return clubName;
    }

    /* Setter dan Getter Club Origin */
    public void setClubOrigin(String clubOrigin) {
        this.clubOrigin = clubOrigin;
    }

    public String getClubOrigin() {
        return clubOrigin;
    }

    /* Setter dan Getter Club Est */
    public void setClubEst(String clubEst) {
        this.clubEst = clubEst;
    }

    public String getClubEst() {
        return clubEst;
    }

}

/* Inherited Class dari class Soccer CLub */
class Player extends SoccerClub {

    /* Private Attribute */
    private String playerName;

    /* Constructor */
    public Player(String playerName) {
        this.playerName = playerName;
    }

    public Player() {

    }

    /* Setter dan Getter Player Name */
    public void setPlayerName(String playerName) {
        this.playerName = playerName;
    }

    public String getPlayerName() {
        return playerName;
    }

    /* Method untuk menampilkan data Player dan Klubnya */
    public void showPlayerData() {
        System.out.println("Club Name : " + this.getClubName());
        System.out.println("Club Origin : " + this.getClubOrigin());
        System.out.println("Club Est : " + this.getClubEst());
        System.out.println("Player Name : " + this.getPlayerName());
    }

}

File: Latihan B/PHP/SoccerClub.php
<?php

class SoccerClub {

    /* Public Attribute (agar dapat diubah oleh class child) */
    public $clubName = "Club Name";
    public $clubOrigin = "Club Origin";
    public $clubEst = "0000";

    /* Constructor */
    public function __construct($clubName = "Club Name", $clubOrigin = "Club Origin", $clubEst = "0000") {
        $this->clubName = $clubName;
        $this->clubOrigin = $clubOrigin;
        $this->clubEst = $clubEst;
    }

    /* Setter dan Getter Club Name */
    public function setClubName($clubName) {
        $this->clubName = $clubName;
    }

    public function getClubName() {
        return $this->clubName;
    }

    /* Setter dan Getter Club Origin */
    public function setClubOrigin($clubOrigin) {
        $this->clubOrigin = $clubOrigin;
    }

    public function getClubOrigin() {
        return $this->clubOrigin;
    }

    /* Setter dan Getter Club Est */
    public function setClubEst($clubEst) {
        $this->clubEst = $clubEst;
    }

    public function getclubEst() {
        return $this->clubEst;
    }

    /* Destructor */
    public function __destruct() {

	}

}

/* Inherited Class dari class SoccerClub */
class Player extends SoccerClub {

    /* Private Attribute */
    private $playerName = "Player Name";

    /* Constructor */
    public function __construct($clubName = "Club Name", $clubOrigin = "Club Origin", $clubEst = "0000", $playerName = "Player Name") {
        $this->clubName = $clubName;
        $this->clubOrigin = $clubOrigin;
        $this->clubEst = $clubEst;
        $this->playerName = $playerName;
    }

    /* Setter dan Getter Player Name */
    public function setPlayerName($playerName) {
        $this->playerName = $playerName;
    }

    public function getPlayerName() {
        return $this->playerName;
    }

    /* Method untuk menampilkan data Player dan Klubnya */
    public function showPlayerData() {
        echo "Club Name : $this->clubName <br />
              Club Origin : $this->clubOrigin <br />
              Club Est : $this->clubEst <br />
              Player Name : $this->playerName <br />";
    }

    /* Destructor */
    public function __destruct() {

	}

}

?>